lab-david/__test__/photo-router.test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const request = require('superagent');
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), { suffix: 'Prom'});
+const debug = require('debug')('cfgram:photo-router.test');
```

Comment:
imports should be ordered as follows:

first party modules
third party modules
your modules